src/types.ts

```diff
@@ -31,7 +31,7 @@ export interface Student {
   is_disabled: boolean;
   display_two_way_communications: boolean;
   display_absences: boolean;
-  can_upload_attachments: string | null;
+  can_upload_attachments: boolean;
```

Comment:
```suggestion
  can_upload_attachments: boolean | null;
```

My ClassCharts has this as null